### ISSUE TITLE
Table- Entire row should get hovered state when tried to access using keyboard navigation

### ISSUE THREAD

**Issue by AnkitaBagale** [razorpay/blade#1996]
When tried to access actionable rows using Tab key, getting each cell focused instead of entire row.
<img width="977" alt="image" src="https://github.com/razorpay/blade/assets/69760792/14f7031b-2e98-4de1-9164-ebcb02dd6b05">

**Comment by AritraLeo** [comment]
Can you please provide a bit more info about the issue I'd like to work on it !

**Comment by saurabhdaware** [comment]
Hey @AritraLeo , Table's most code is in this directory - https://github.com/razorpay/blade/blob/master/packages/blade/src/components/Table/TableBody.web.tsx

@chaitanyadeorukhkar can you take a look at see what should be the keyboard behaviour in selectable table? I couldn't find a way to select an item with keyboard navigations

**Comment by AritraLeo** [comment]
Thanks @saurabhdaware

**Comment by aditya2410-U** [comment]
is this issue resolved , if not can i pick this?
